static/js/top_left_corner.js

```diff
@@ -105,6 +105,7 @@ export function narrow_to_recent_topics() {
     remove($(".top_left_private_messages"));
     remove($(".top_left_starred_messages"));
     remove($(".top_left_mentions"));
+    $("#resume_reading_banner").hide();
```

Comment:
I think it's better to have this at the start of `recent_topics.show()` instead of tracking down all its callers.

Reply:
Yeah, I agree that's a much better place.  Moved it there.